packages/salesforcedx-vscode-core/src/context/workspaceContext.ts

```diff
@@ -46,7 +46,6 @@ export class WorkspaceContext {
       // error reported by setupWorkspaceOrgType
       console.error(e)
     );
-    await ConfigAggregatorProvider.getInstance().reloadConfigAggregators();
```

Comment:
this is now being called in the utils package where the listener first reacts to the config file change, instead of here in the workspaceContext module where the event is handled